rosette/src/Makefile

```diff
@@ -24,7 +24,7 @@
 
 ARCH		= linux
 
-CPPFLAGS	= -I ../h -I ../h/sys -DARCH_INC=\"$(ARCH).h\" -m32
+CPPFLAGS	= -I ../h -I ../h/sys -DARCH_INC=\"$(ARCH).h\" -m32 -Wno-write-strings -Wno-invalid-offsetof -Wno-delete-incomplete -Wno-deprecated-declarations
```

Comment:
> tfw no -Wall

Reply:
I'm trying to balance getting this stuff to build properly with the amount of work that would be required to refactor it all.

Reply:
Some warnings are better than none.